### PR TITLE
Uml 1200 : dashboard grey panels

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-1200

Fixes UML-1200

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-api/seeding/dynamodb.py [hunk 1]
```diff
@@ -126,6 +126,20 @@
...
+            'ActorId': '9',
+            'Added': '2021-04-22T15:01:11.548361Z',
+            'UserId': 'bf9e7e77-f283-49c6-a79c-65d5d309ef77'
+    },
+    {
+        'Id': '1600be0d-727c-41aa-a9cb-45857a73ba4f',
+        'SiriusUid': '700000000252',
+        'ActorId': '43',
+        'Added': '2021-04-23T11:44:11.324804Z',
+        'UserId': 'bf9e7e77-f283-49c6-a79c-65d5d309ef77'
```
Comment (GemTay) [inline service-api/seeding/dynamodb.py:142]: are these seeding changes intended to be merged?

**Review by GemTay — CHANGES_REQUESTED** [review]
Somethings still not working correctly, I've debugged and checked that the 2 "Babara Suzanne Gilson" LPAs are grouped together correctly in the sorting & grouping functions, so somewhere is breaking them apart on the UI
image

File: service-api/seeding/dynamodb.py (hunk 1 above)
Comment (SeemaMenon) [reply service-api/seeding/dynamodb.py:142]: yes, to have multiple LPA's added for the user with different statuses.

File: service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+    <div class="govuk-summary-list__row govuk-!-margin-bottom-0 govuk-!-margin-top-0">
         <dt class="govuk-summary-list__key govuk-!-margin-bottom-0 govuk-!-width-one-third">
             {% if actorActive %}
                 <h2 class="govuk-heading-m govuk-!-margin-bottom-0">{{ caseSubtype }}</h2>
             {% else %}
                 <h2 class="govuk-heading-m govuk-!-margin-bottom-0 lpa-secondary-text-colour">{{ caseSubtype }}</h2>
             {% endif %}
         </dt>
-        <dd class="govuk-summary-list__value govuk-!-padding-top-4 govuk-!-padding-bottom-4 govuk-!-margin-bottom-0">
+        <dd class="govuk-summary-list__value govuk-!-padding-top- govuk-!-padding-bottom-4 govuk-!-margin-bottom-0">
```
Comment (GemTay) [inline service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig:10]: `govuk-!-padding-top-` is missing a number

Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig]: added now

**Review by GemTay — APPROVED** [review]

**Pull request merged**